bin/spark-evaluate.js

```diff
@@ -43,6 +48,40 @@ const createPgClient = async () => {
   return pgClient
 }
 
+const pgClient = await createPgClient()
+const cancelStuckTransactions = new CancelStuckTransactions({
```

Comment:
Can you please move this to a new file in `lib`? I'd like to keep `bin` files only a thin wrappers around stuff imported from elsewhere.